src/style.css

```diff
@@ -0,0 +1,107 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 800;
+}
+
+.container {
+  margin-top: 20px;
+}
+
+.form {
+  position: relative;
+  height: 70px;
+}
+
+.form:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.image {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  cursor: text;
+}
+
+.search {
+  outline: 0;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
```

Comment:
Use CSS variables for repeatable colors. Move these variables to `:root` selector.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)